examples/dishwasher-app/silabs/data_model/dishwasher-wifi-app.matter

```diff
@@ -2593,7 +2593,6 @@ endpoint 2 {
   }
 }
 endpoint 3 {
-  device type device_energy_management = 1293, version 1;
```

Comment:
Why did this change?  This looks odd.

Is the .zap file referencing this device type by name and the name changed?

Reply:
The `<Name>` changed, not the `<TypeName>`. I would guess Zap should use TypeName instead of Name?
Let me try keeping the name the same and see if it changes anything.

Reply:
Alright, so if I keep the name the same, it doesn't change this. So, this is being consumed in the zap regen somehow. I pushed a commit without those changes.